src/umpire/ResourceManager.cpp

```diff
@@ -709,4 +710,18 @@ ResourceManager::getZeroByteAllocator()
   return m_allocators_by_name[s_zero_byte_pool_name];
 }
 
+std::shared_ptr<op::MemoryOperation>
+ResourceManager::getOperation(
+    const std::string& operation_name,
+    Allocator src_allocator,
```

Comment:
Will the following suggestion work?

```suggestion
    const Allocator& src_allocator,
```

Reply:
Should do, let me try 😄